### PR TITLE
styling

### PR DESCRIPTION
**Pull request by rabe200**
lgty?

### CODE REVIEW

File: src/components/StyledImageContainer.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+const StyledImageContainer = styled.div`
+  background-color: ghostwhite;
+  border-radius: 20px;
+  margin-bottom: 20px;
+  position: relative;
+  margin-top: 50px;
+  border: 0px solid orange;
+  filter: drop-shadow(0px 0px 3px white);
```
Comment (Satttoshi) [inline src/components/StyledImageContainer.js:10]: add width based on image width. wrap title if it's too long, the favorite button is getting outa bounce here.

attachement

**Review by Satttoshi — CHANGES_REQUESTED** [review]
please check change-request. rest LGTM :rocket: gj

**Comment by rabe200** [comment]
bug fixed

**Comment by Satttoshi** [comment]
LGTM :rocket:

**Pull request merged**